docs/split_keyboards.md

```diff
@@ -14,22 +14,22 @@ keyboard.uart_pin = board.SCL
 Useful config options:
 ```python
 keyboard.split_flip = True  # If your boards are identical but one is flipped, this option is for you 
-keyboard.split_offsets = [6, 6, 6, 6]  # This is the how many keys are on each column on the "Master" half
+keyboard.split_offsets = [6, 6, 6, 6]  # This is the how many keys are on each column on the target" half
```

Comment:
missed a "

Reply:
This is correct. The "target" is the master half. This terminology can be found in fibre networks.

Reply:
Don't know what I thought I was responding to. Fixing. Thanks.